vars_template.yaml

```diff
@@ -28,6 +28,7 @@ dns_zone_name: str              # Cloud DNS zone name for the cluster
 # Common Legion variables
 ########################################################
 legion_helm_repo: str           # Helm repository url with Legion charts 
+istio_helm_repo: str            # Helm repository url with Istio charts 
```

Comment:
as I understand we should also add this parameter to legion-profiles and open appropriate PR

Reply:
yes, already merged